### ISSUE TITLE
Stevia creating issue with main project auto layout library

### ISSUE THREAD

**Issue by r-a-o** [Yummypets/YPImagePicker#603]
Dynamically linked dependency, Stevia, is creating a lot of class conflicts with EasyPeasy which is another popular autolayout library which I use in my project. 
Is there a way I can remove dynamic linking of Stevia and still make YPImagePicker work?

**Comment by bleakas** [comment]
same problem

**Comment by alloc33** [comment]
same!

**Comment by NikKovIos** [comment]
Nope. YPImagePicker makes it layout with Stevia. Stevia is necessary for YPImagePicker.
